app/elements/elements.html

```diff
@@ -32,11 +32,6 @@
 <link rel="import" href="../bower_components/paper-input/paper-textarea.html">
 <link rel="import" href="../bower_components/paper-button/paper-button.html">
 
-<!-- Uncomment next block to enable Service Worker Support (2/2) -->
-
-<link rel="import" href="../bower_components/platinum-sw/platinum-sw-cache.html">
```

Comment:
Aah okay. We could use it for offline caching of the frontend, but realistically we'll need api connectivity anyway